### PR TITLE
Updated KUBE_GIT_VERSION_FILE and bumped prod RELEASE for 1.20-2

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
Hold until release

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kschumy** [comment]
/hold

Until release is ready to be cut

**Review by TerryHowe — COMMENTED** [review]
/lgtm

File: projects/kubernetes/kubernetes/1-20/KUBE_GIT_VERSION_FILE [hunk 1]
```diff
@@ -1,6 +1,6 @@
-KUBE_GIT_COMMIT='e87da0bd6e03ec3fea7933c4b5263d151aafd07c'
```
Comment (TerryHowe) [inline projects/kubernetes/kubernetes/1-20/KUBE_GIT_VERSION_FILE:1]: Again, I feel this file should be modified in the dev release PR, no?

**Comment by kschumy** [comment]
Deleting in favor of https://github.com/aws/eks-distro/pull/413

**Pull request closed without merging**